Documentation/network/clustermesh/clustermesh.rst

```diff
@@ -80,16 +88,12 @@ Scaling Limitations
  * Helm option ``--set clustermesh.maxConnectedClusters=511``
  * ``cilium install`` option ``--set clustermesh.maxConnectedClusters=511``
 
-* This option controls the bit allocation of numeric identities and will affect the number of
-  identities that can be allocated per cluster:
+.. note::
 
-+------------------------+------------+----------+----------+
-| MaxConnectedClusters   | Maximum cluster-local identities |
-+========================+============+==========+==========+
-| 255 (default)          | 65535                            |
-+------------------------+------------+----------+----------+
-| 511                    | 32767                            |
-+------------------------+------------+----------+----------+
+   This option controls the bit allocation of numeric identities and will affect the maximum number
```

Comment:
```suggestion
   ``MaxConnectedClusters`` controls the bit allocation of numeric identities and will affect the maximum number
```
non-blocking suggestion to make the note easier to read

Reply:
Please resolve this convo to get the PR merged.